examples/coagents-starter/README.md

```diff
@@ -6,14 +6,23 @@ This example contains a simple starter project.
 
 ## Running the Agent
```

Comment:
This repo comes with two different agents, one written in Python and one in JavaScript.